app/controllers/demo_controller.rb

```diff
@@ -1,4 +1,22 @@
 class DemoController < ApplicationController
+  def reset
+    set_current_user({beverage: 0})
+    session[:percentage] = 0
+    Flipper.disable_percentage_of_actors(:slow_roll)
+    Flipper.disable_percentage_of_time(:slow_roll)
+    Flipper.disable(:demo)
+    Flipper.disable(:demo_actor)
+    Flipper.disable(:demo_group)
+    Flipper.disable(:slow_roll)
+
+    sleep 1
```

Comment:
😴 ?

Reply:
For some reason, I wasn't seeing the flag changes applied and pages refreshed appropriately. I'll see if some other changes have fixed it so I can remove that.